### ISSUE TITLE
Update docs for react_router using SSR

### ISSUE THREAD

**Issue by jeremylynch** [shakacode/react_on_rails#1632]
The current docs for react-router are outdated and incomplete: https://www.shakacode.com/react-on-rails/docs/javascript/react-router/

It would be great to update the following: 

- Update to use React router v7
- Update redux to use `configureStore` rather than `createStore` (deprecated)
- Show recommended way of setting up redux store when using react_on_rails

And a full end-to-end example of setting react_router with SSR would be fantastic!

**Comment by justin808** [comment]
You are right! @jeremylynch any chance that you could add a PR?

BTW -- nice profile picture! I'm a surfer as well.

**Comment by jeremylynch** [comment]
Thanks Justin! 

I would happily do so, but I couldn't get this to work despite much trial and error. 

Are there any example apps which show a working version of this? If I can find one I can update the docs

**Comment by justin808** [comment]
Yes, see https://github.com/shakacode/react-webpack-rails-tutorial

**Comment by jeremylynch** [comment]
Thanks Justin! I will review
